tests/test_filter_classes.py

```diff
@@ -249,6 +273,24 @@ def test_filter_sos_process(impulse):
     npt.assert_allclose(res.time, np.atleast_2d(desired))
 
 
+def test_filter_sos_process_complex(impulse_complex):
+    sos = np.array([[1, 1/2, 0, 1, 0, 0]])
+    filt = fo.FilterSOS(sos, impulse_complex.sampling_rate)
+    coeff = np.array([[1, 1/2, 0], [1, 0, 0]])
```

Comment:
Why are you initializing twice? According to the docs the shape of the coefficients should be `(1, 1, 6)` did that not work?

Reply:
It was a copy and paste from the test for the non complex data, to be honest :) ... I removed. Also in the non complex test. Should I also test it with coefficients (1, 1, 6)?

Reply:
I tried it and filt.coefficients have shape `(1,1,6)` in both cases. but maybe inputting the coefficients according to the docs is less confusing, i.e., using `sos = np.array([[[1, 1/2, 0, 1, 0, 0]]])`